types/actioncable/index.d.ts

```diff
@@ -1,13 +1,15 @@
-// Type definitions for ActionCable
-// Project: https://github.com/rails/rails/tree/master/actioncable
+// Type definitions for ActionCable 5.2
+// Project: https://github.com/rails/rails/tree/master/actioncable/app/assets/javascripts
 // Definitions by: Vincent Zhu <https://github.com/zhu1230>
+//                 Jared Szechy <https://github.com/szechyjs>
 // Definitions: https://github.com/zhu1230/DefinitelyTyped
+// TypeScript Version: 2.3
 
 declare module ActionCable {
   interface Channel {
     unsubscribe(): void;
     perform(action: string, data: {}): void;
-    send(data: Object): boolean;
+    send(data: any): boolean;
```

Comment:
Per the advice in [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes) I changed it to `any`, looking at the actioncable source it calls `JSON.stringify` on the data, which leads me to believe that it would work with anything. If this is not the case I can certainly change it back.

Reply:
If it's genuinely open-ended, then `any` is the right choice.  Thanks for confirming!